ui/src/app/shared/utils.ts

```diff
@@ -120,5 +120,13 @@ export const Utils = {
             return false;
         }
         return wf.status.phase === 'Running';
+    },
+
+    setCachedNamespace(value: string): void {
```

Comment:
minor - could this be a getter/setting method, 

```public get cachedNamespace() {..}```

Reply:
may `currentNamespace`?

Reply:
> maybe `currentNamespace`?

I'm ok with either one. will change it to `current`.